client/components/product-card/index.jsx

```diff
@@ -55,7 +58,14 @@ const ProductCard = ( {
 					) }
 				</div>
 			</div>
-			{ description && <p className="product-card__description">{ description }</p> }
+			<div className="product-card__description">
+				{ hasManageSubscriptionLink && (
+					<p>
+						<a href="/my-plan">{ translate( 'Manage Subscriptions' ) }</a>
```

Comment:
This url takes me to a 404.

Reply:
I think this will have to be replaced by some value provided by the parent block. What do you think @tyxla?

Reply:
Indeed, this will be replaced with another parameter as we discussed. The URL will be coming from a purchase object which is not yet available here. I'll be taking care of that in another PR, so this is good enough for now.

Reply:
Thanks for the explanation!